secure_message/authentication/jwt.py

```diff
@@ -2,22 +2,25 @@
 Module to generate jwt token
 """
 
+from flask import current_app
 from jose import jwt
-from app import settings
-
-JWT_ALGORITHM = settings.SM_JWT_ALGORITHM
-JWT_SECRET = settings.SM_JWT_SECRET
 
 
 def encode(data):
     """
     Function to encode python dict data
     """
+    JWT_ALGORITHM = current_app.config['SM_JWT_ALGORITHM']
```

Comment:
Could be lower cased too

Reply:
Does this affect the Jenkins config? If so we don't want to change the case as it will affect Jenkins